src/renderer/views/Watch/Watch.js

```diff
@@ -299,7 +305,15 @@ export default Vue.extend({
           })
 
           this.videoPublished = new Date(result.videoDetails.publishDate.replace('-', '/')).getTime()
-          this.videoDescription = result.player_response.videoDetails.shortDescription
+          try {
+            // workaround for description localization
+            const descriptionLines = result.response.contents.twoColumnWatchNextResults.results.results.contents[1].videoSecondaryInfoRenderer.description.runs
+            this.videoDescription = descriptionLines?.join('\n') ?? ''
```

Comment:
Oh whoops, sorry missed that part. To keep it as a one liner you could change it to this:
```suggestion
            this.videoDescription = descriptionLines?.map(line => line.text).join('\n') ?? ''
```

Or go back to your old solution, as that will definitely work.